enclave/sgx/revocationinfo.c

```diff
@@ -25,6 +32,12 @@ oe_result_t oe_get_revocation_info(oe_get_revocation_info_args_t* args)
     if (!args)
         OE_RAISE(OE_INVALID_PARAMETER);
 
+    /*Allocate a default revocation-info-arg size*/
+    if(sizeof(out) < DEFAULT_ARGS_SIZE)
+    {
+        out = *((oe_get_revocation_info_args_t *)oe_malloc(DEFAULT_ARGS_SIZE));
```

Comment:
I don't follow. Why does changing the size of out change anything? The fields of out are passed to `oe_get_revocation_info_ocall` individually, the whole structure isn't used anywhere.

Isn't the issue that the individual buffers (such as `tcb_info` and `tcb_issuer_chain`) are null the first time around and so it takes an ocall just to get the sizes. So if they had preallocated buffers for each field then it could be done in one ocall instead of two.

Reply:
I agree. I'll make these changes. Thanks you.

Reply:
This file, enclave/sgx/revocationinfo.c will be removed from this PR.